tests/debug/lldb.rs

```diff
@@ -0,0 +1,92 @@
+#![allow(dead_code)]
+
+use anyhow::{bail, format_err, Result};
+use filecheck::{CheckerBuilder, NO_VARIABLES};
+use std::env;
+use std::io::Write;
+use std::process::Command;
+use tempfile::NamedTempFile;
+
+fn lldb_with_script(args: &[&str], script: &str) -> Result<String> {
+    let mut me = std::env::current_exe().expect("current_exe specified");
+    me.pop(); // chop off the file name
+    me.pop(); // chop off `deps`
+    me.push("wasmtime");
+    let wasmtime_path = me.to_str().unwrap();
```

Comment:
If the `Command` type is created directly I think the `to_str()` conversion and the `lldb_args` vec can both be elided in favor of just calling the methods on `Command` to append arguments.